install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -475,21 +475,23 @@ data:
 {{- if hasKey .Values.nodePort "directRoutingDevice" }}
   direct-routing-device: {{ .Values.nodePort.directRoutingDevice | quote }}
 {{- end }}
-{{- if hasKey .Values.nodePort "mode" }}
-  node-port-mode: {{ .Values.nodePort.mode | quote }}
-{{- end }}
-{{- if hasKey .Values.nodePort "algorithm" }}
-  bpf-lb-algorithm: {{ .Values.nodePort.algorithm | quote }}
-{{- end }}
 {{- if .Values.nodePort.enableHealthCheck }}
   enable-health-check-nodeport: {{ .Values.nodePort.enableHealthCheck | quote}}
-{{- end }}
-{{- if hasKey .Values.nodePort "acceleration" }}
-  node-port-acceleration: {{ .Values.nodePort.acceleration | quote }}
 {{- end }}
   node-port-bind-protection: {{ .Values.nodePort.bindProtection | quote }}
   enable-auto-protect-node-port-range: {{ .Values.nodePort.autoProtectPortRange | quote }}
 {{- end }}
+{{- if hasKey .Values "loadBalancer" }}
+{{- if hasKey .Values.loadBalancer "mode" }}
+  node-port-mode: {{ .Values.loadBalancer.mode | quote }}
```

Comment:
I might miss some context here, can we rename node-port-mode to bpf-lb-mode now ?

Reply:
Right, both are an alias to `bpf-lb-mode` and the other one to `bpf-lb-acceleration`. I left them as is here to avoid any potential breakage when upgrading from 1.8.

Reply:
oh right :), thanks for clarification :100: